### ISSUE TITLE
Training with COCO dataset

### ISSUE THREAD

**Issue by NRPerson** [clovaai/BESTIE#19]
When training with coco dataset, I can't achieve the performance in your paper anyway. If it is not open source for the time being, can you first disclose some parameters or experimental details, such as the data enhancement strategies, learning rate, loss function weight, etc.

**Comment by wsis-seg** [comment]
can you give me the coco datasets code? emails:[[526078694@qq.com](mailto:526078694@qq.com)]
